dart-templates/serialization/native/class.mustache

```diff
@@ -247,11 +247,11 @@ class {{{classname}}} {{#interfaces}}{{#-first}}implements {{/-first}}{{.}}{{^-l
   @override
   String toString() => '{{{classname}}}[{{#vars}}{{{name}}}=${{{name}}}{{^-last}}, {{/-last}}{{/vars}}]';
 
-  Map<String, dynamic> toJson([List<String>? keys]) {
+  Map<String, dynamic> toJson([Iterable<String>? keys]) {
```

Comment:
Out of curiosity : when do we need something other than a List ? Thanks

Reply:
If we were to extract items from a `List` or other `Iterable` with a method that returns an `Iterable` or we want to provide a `Set` to make sure we don't have duplicates.